llvm/tools/llvm-exegesis/lib/Target.h

```diff
@@ -86,6 +86,14 @@ class ExegesisTarget {
                 ArrayRef<const char *> ValidationCounters,
                 const pid_t ProcessID = 0) const;
 
+  virtual bool checkOpcodeSupported(int Opcode,
+                                    const MCSubtargetInfo &SI) const {
+    return true;
+  }
+
+  // Find register by name, NoRegister if not found.
+  virtual unsigned findRegisterByName(const StringRef RegName) const = 0;
```

Comment:
Why can't this be non-virtual and use `RegNameToRegNoMapping` ?

Reply:
Create default implementation for all platforms. Thank you!